match/lib/match/encrypt.rb

```diff
@@ -93,6 +93,17 @@ def crypt(path: nil, password: nil, encrypt: true)
       success = system(command.join(' '))
 
       UI.crash!("Error decrypting '#{path}'") unless success
+
+      # On non-Mac systems it might take some time for the file to actually be there (#11182)
```

Comment:
Interesting, is that an issue in the `openssl` command? Is there a flag to make it a blocking call?

Reply:
I have no idea tbh. Found the solution by looking at the follow up problem.Should I investigate this further instead of this workaround?

Reply:
(As this is only run on non-Mac systems I would prefer to just have this merged and investigate later, maybe also switch to an openssl gem [that I think is already in use in other places])